lib/jobs/basejob.h

```diff
@@ -67,6 +67,7 @@ namespace QMatrixClient
                 , UnsupportedRoomVersionError
                 , NetworkAuthRequiredError
                 , UserConsentRequiredError
+                , UnknownError // Unknown room or other item (M_UNKNOWN)
```

Comment:
Can we please make it `UnknownObjectError` or something? `UnknownError` looks really ambiguous.